libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -77,7 +77,7 @@ internal class MapboxTripSession(
 
     private companion object {
         private val TAG = Tag("MbxTripSession")
-        private const val MIN_NEXT_WAYPOINT_INDEX = 1
+        private const val INDEX_OF_ORIGIN_LOCATION = 1
```

Comment:
Origin location is actually the index 0.

What we're looking for here, is marking that when the reroute happens, we always want to ignore the origin.

```suggestion
        private const val INDEX_OF_INITIAL_LEG_TARGET = 1
```

Reply:
Great name, thank you 👍